pkg/ebpf/c/tracee.bpf.c

```diff
@@ -6328,9 +6248,4 @@ SEC("tc")
 int tc_ingress(struct __sk_buff *skb)
 {
     return tc_probe(skb, true);
-}
-
-char LICENSE[] SEC("license") = "GPL";
-#ifndef CORE
-int KERNEL_VERSION SEC("version") = LINUX_VERSION_CODE;
-#endif
+}
```

Comment:
nit: The newline is gone

Reply:
Yep, if this is something to be enforced we should have in .clang-format file as well… Checking...

Reply:
There does not seem to be a way to enforce that style, so I'm afraid it is a no-go (to have empty line after last line).